src/select.jsx

```diff
@@ -369,7 +369,8 @@ var classBase = React.createClass({
           tabIndex={-1}
           aria-hidden={true} >
             {React.Children.map(this.props.children, function (child, index) {
-              return <option key={index} value={child.props.value}>{child.props.children}</option>
+              var value = child.props.value || child.props.children;
+              return <option key={index} value={child.props.value}>{value}</option>
```

Comment:
I am mistaken `value={child.props.value}` is correct. I still think we want the contents to be `>{child.props.children<` otherwise the content won't show up properly.